wradlib/georef/vector.py

```diff
@@ -479,6 +479,6 @@ def get_centroid(polyg):
         x and y coordinate of the centroid
 
     """
-    if not type(polyg) == ogr.Geometry:
+    if type(polyg) is not ogr.Geometry:
```

Comment:
These `is not` checks should be replaced with isinstance checks as well I think. Or do we want to be absolutely sure that `trg` is of type `Geometry` and not a subtype of `Geometry`?
```suggestion
    if not isinstance(polyg, ogr.Geometry):
```

Reply:
Obviously missed those. Let's try it.